moveit_commander/src/moveit_commander/move_group.py

```diff
@@ -578,10 +578,10 @@ def set_support_surface_name(self, value):
         """ Set the support surface name for a place operation """
         self._g.set_support_surface_name(value)
 
-    def retime_trajectory(self, ref_state_in, traj_in, velocity_scaling_factor):
+    def retime_trajectory(self, ref_state_in, traj_in, velocity_scaling_factor=1.0, acceleration_scaling_factor=1.0, algorithm=""):
```

Comment:
Why don't you specify `"iterative_time_parameterization"` as the default here?

Reply:
Updated to specify `"iterative_time_parameterization"`.